src/ucp/core/ucp_ep.h

```diff
@@ -428,6 +428,8 @@ typedef struct {
     ucs_ptr_map_key_t        remote_ep_id; /* Remote EP ID */
     ucp_err_handler_cb_t     err_cb; /* Error handler */
     ucp_ep_close_proto_req_t close_req; /* Close protocol request */
+    /* Endpoint name for tracing and analysis */
+    char                     name[UCP_ENTITY_NAME_MAX];
```

Comment:
we cannot increase endpoint size even when names/vfs are not used, because it's not scalable

Reply:
maybe we do not have to increase it even when name is defined? let it be just pointer

Reply:
context name + worker name + id (long) should be enough to uniquely identify ep. I don't think we need char* all together.

Reply:
@shamisp in some cases it's useful to associate application connection name (for example MPI rank, SHMEM PE, Spark executor ID) with the endpoint object. Do you suggest passing "long id" instead of "char *name" in the API?

Reply:
@shamisp 
What do you think about having user-defined ep name in case of enabled debug data? EP memory address value will be used as name, otherwise.

Reply:
@shamisp ?